lib/index.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
     'use-brace-expansion': require('./rules/use-brace-expansion'),
     'use-ember-data-rfc-395-imports': require('./rules/use-ember-data-rfc-395-imports'),
     'use-ember-get-and-set': require('./rules/use-ember-get-and-set'),
+    'use-waiters-in-module-scope': require('./rules/use-waiters-in-module-scope'),
```

Comment:
I think I'd prefer `require-test-waiters-in-module-scope` as the rule name. Why? 

* The rule is not just about *using* the test waiters, it's more the requirement about how to use them
* We try to use `test` in all test-related rule names for clarity

Reply:
Sure. Naming is hard yada yada

Reply:
Updated the name to `no-invalid-test-waiters`